assayist/processor/base.py

```diff
@@ -290,11 +290,10 @@ def claim_file(self, base_dir, path_in_base_dir):
 
         :param str base_dir: the base directory to claim a file from
         :param str path_in_base_dir: the path to the file in the base directory to claim
-        :raises RuntimeError: when path_in_base_dir is the root directory or the path to the
-            base_dir is not a directory
+        :raises RuntimeError: when the path to the base_dir is not a directory
         """
         if path_in_base_dir == '/':
-            raise RuntimeError('You cannot claim the root directory of a container')
+            return
```

Comment:
Fyi, this will be raised for every base layer container such as build 798524. Noop seems like a safe alternative.